plugins/analytics/public/dockerhub.py

```diff
@@ -0,0 +1,201 @@
+import json
+import logging
+import re
+
+import requests
+from core import taskmanager
+from core.config.config import yeti_config
+from core.schemas import observable, task
+from core.schemas.observable import Observable, ObservableType
+
+
+class DockerHubApi(object):
```

Comment:
```suggestion
class DockerHubApi:
```

Especially since you'll never instantiate this (all methods are static)